src/Adapters/Storage/Mongo/MongoTransform.js

```diff
@@ -330,18 +330,11 @@ const parseObjectToMongoObjectForCreate = (className, restCreate, schema) => {
 // Main exposed method to help update old objects.
 const transformUpdate = (className, restUpdate, parseFormatSchema) => {
   let mongoUpdate = {};
-  let acl = addLegacyACL(restUpdate)._acl;
-  if (acl) {
-    mongoUpdate.$set = {};
-    if (acl._rperm) {
-      mongoUpdate.$set._rperm = acl._rperm;
-    }
-    if (acl._wperm) {
-      mongoUpdate.$set._wperm = acl._wperm;
-    }
-    if (acl._acl) {
-      mongoUpdate.$set._acl = acl._acl;
```

Comment:
An alternative fix would be to remove the ._acl on line 334 and keep the original code, i.e.
```
  let acl = addLegacyACL(restUpdate);
  if (acl) {
    mongoUpdate.$set = {};
    if (acl._rperm) {
      mongoUpdate.$set._rperm = acl._rperm;
    }
    if (acl._wperm) {
      mongoUpdate.$set._wperm = acl._wperm;
    }
    if (acl._acl) {
      mongoUpdate.$set._acl = acl._acl;
    }
  }
```
I felt that was pointless though as _rperm and _wperm are set elsewhere and thought there must have been a reason line 334 was changed in the first place to reference _acl